### DISCUSSION TITLE
Unable to assign user to a custom role

### DISCUSSION THREAD

**Discussion by debeardjr** [PiranhaCMS/piranha.core#1643] [Q&A]
Hello,
I am using SQL Server Identity and have created a role named Content Manager. When I select this role for a user, I receive an error, **Role CONTENT MANAGER does not exist.**

![image](https://user-images.githubusercontent.com/12800061/120344640-c4256e80-c2c7-11eb-9659-f995d3b8666a.png)

**Comment by tidyui** [comment]
You should open issue for this instead! In the meantime please try to remove the space in the role name as it’s highly likely this is causing the error.

Regards
